packages/appstore/src/modules/traders-hub/slick.scss

```diff
@@ -0,0 +1,102 @@
+/* Slider */
+
+.slick-slider {
+    position: relative;
+    display: block;
+    box-sizing: border-box;
+    -webkit-touch-callout: none;
+    -webkit-user-select: none;
+    -khtml-user-select: none;
+    -moz-user-select: none;
+    -ms-user-select: none;
+    user-select: none;
+    -ms-touch-action: pan-y;
+    touch-action: pan-y;
+    -webkit-tap-highlight-color: transparent;
+}
+.slick-list {
+    position: relative;
+    overflow: hidden;
+    display: block;
+    margin: 0;
+    padding: 0;
+
+    &:focus {
+        outline: none;
+    }
+
+    &.dragging {
+        cursor: pointer;
+        cursor: hand;
+    }
+}
+.slick-slider .slick-track,
+.slick-slider .slick-list {
+    -webkit-transform: translate3d(0, 0, 0);
+    -moz-transform: translate3d(0, 0, 0);
+    -ms-transform: translate3d(0, 0, 0);
+    -o-transform: translate3d(0, 0, 0);
+    transform: translate3d(0, 0, 0);
+}
+
+.slick-track {
+    position: relative;
+    left: 0;
+    top: 0;
+    display: block;
+    margin-left: auto;
+    margin-right: auto;
+
+    &:before,
+    &:after {
+        content: '';
+        display: table;
+    }
+
+    &:after {
+        clear: both;
+    }
+
+    .slick-loading & {
+        visibility: hidden;
+    }
+}
+.slick-slide {
+    float: left;
+    height: 100%;
+    min-height: 1px;
+    // padding-inline: 1.2rem;
+
+    [dir='rtl'] & {
+        float: right;
+    }
+    img {
+        display: block;
+    }
+    &.slick-loading img {
+        display: none;
+    }
+
+    display: none;
+
+    &.dragging img {
+        pointer-events: none;
+    }
+
+    .slick-initialized & {
+        display: block;
+    }
+
+    .slick-loading & {
+        visibility: hidden;
+    }
+
+    .slick-vertical & {
+        display: block;
+        height: auto;
+        border: 1px solid transparent;
+    }
+}
+.slick-arrow.slick-hidden {
+    display: none;
+}
```

Comment:
Same here

Reply:
@aizad-deriv this property will be applied only for element which has slick-arrow and slick-hidden classes at the same time, so in my opinion it will not affect other elements.

Reply:
@aizad-deriv I can't delete file because there are styles for slider arrows. But I deleted styles for dots (default pagination) because we don't use it here

Reply:
@sergei-deriv but at least can we just rename the classes and follow what we have in the code? 
I can see there's a [className](https://react-slick.neostack.com/docs/api#className) within the docs where you can overwrite CSS class for inner slider div and [dotsClass](https://react-slick.neostack.com/docs/api#dotsClass) to overwrite the dots class.

Reply:
@aizad-deriv yes, you are almost right. I can change dotsClass, but I'm using our Pagination with own classes, so I don't need this. About className for innerSlider, it's just add your class to the parent div and that's all, every slide have `slick-slide` class so I've added this inside of `wallet-cards-carousel` class in scss file.

Reply:
@sergei-deriv couldn't we move this to the components package as well?